### DISCUSSION TITLE
Menu导航菜单选中样式

### DISCUSSION THREAD

**Discussion by WT-Road** [ant-design/ant-design#38834] [General]
V4二级菜单选中样式(一级菜单有选中颜色)
![截屏2022-11-22 10 11 32](https://user-images.githubusercontent.com/14371925/203200265-7e768d02-f985-4d24-ada8-a52bba7d22f6.png)

V5二级菜单选中样式(一级菜单没有选中颜色)
![截屏2022-11-22 10 14 14](https://user-images.githubusercontent.com/14371925/203200606-ad91be3f-2622-4b8f-8993-36258d89f30a.png)

一级菜单下没有子菜单时,V4和V5都有选中样式
所以目前V5就出现选中样式不统一
以下截图是V5
想知道这样设计的原则是什么?
![image](https://user-images.githubusercontent.com/14371925/203201030-923e51a1-724f-46e0-9db5-fbfbbc8d7d08.png)
![image](https://user-images.githubusercontent.com/14371925/203201067-6a7f70ba-ab45-436d-b759-5578f1ebfae1.png)

**Comment by afc163** [comment]
样式迁移的时候丢了，是 bug 。
